### PR TITLE
Enable GitHub workflows for tests

### PR DESCRIPTION
**Pull request by plettich**

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Python test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (cornelinux) [inline .github/workflows/pytest.yml]: Can we change this to 18.04?
So this would match with our appliance.

File: .github/workflows/pytest.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
```
Comment (cornelinux) [inline .github/workflows/pytest.yml]: We probably only need 3.5 (16.04) and 3.6 (18.04). If this helps to ease things.

File: tox.ini [hunk 3]
```diff
@@ -1,5 +1,12 @@
 [tox]
-envlist = py27,py36
+envlist = py27,py35,py36,py37
```
Comment (cornelinux) [inline tox.ini]: again, would 3.5 and 3.6 be enough at the moment?

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (plettich) [reply .github/workflows/pytest.yml]: I'll check that out

File: .github/workflows/pytest.yml (hunk 2 above)
Comment (plettich) [reply .github/workflows/pytest.yml]: I'd rather keep 2.7 just for testing purposes. 3.7 is removed.

File: tox.ini (hunk 3 above)
Comment (plettich) [reply tox.ini]: see above

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (plettich) [reply .github/workflows/pytest.yml]: fixed in e2fb651

**Pull request merged**